legal-api/src/legal_api/reports/report.py

```diff
@@ -363,6 +372,7 @@ class ReportMeta:  # pylint: disable=too-few-public-methods
     }
 
     entity_description = {
-        'CP': 'cooperative',
-        'BC': 'BC Benefit Company'
+        Business.LegalTypes.COOP.value: 'cooperative',
```

Comment:
Everything else is init capped, other than cooperatives? Should it be consistent?

Reply:
@kialj876 is going to open a tech debt tix for this. With that, it's resolved in the is PR.

Reply:
https://github.com/bcgov/entity/issues/5088 made for this